### DISCUSSION TITLE
IOS App?

### DISCUSSION THREAD

**Discussion by powerdollkirby** [stoatchat/discussions#791] [Q&A]
i wonder if there's any plans for an ios app, even if it's through donations just to keep it up

**Comment by amycatgirl** [comment]
There is one in the works, it's available [here](https://github.com/revoltchat/ios)

If you want to try it on testflight, do note that the app is missing many features that the web client provides, and it might be very buggy from time to time, that being said, test at your own discretion.

**Reply by powerdollkirby** [reply]
yeah thanks for that

looking good already, but it keeps crashing when doing anything, i reported it, i might have to wait and continue using the web client
